nss-crystal-cobras/WorkforceManagement
# Almost done minus the insert and update.

Ticket 4.1( https://github.com/nss-crystal-cobras/WorkforceManagement/issues/22 ) View Editable Employee

Still working on edit I am currently able to display and see everything but my SQL inserts and Updates are not updating the database.

## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made

git fetch all 
git checkout EmpEditJD
Check it out using the edit on each employee.


# Checklist:
